utils/build/docker/python/flask/app.py

```diff
@@ -531,7 +531,8 @@ def consume_kafka_message():
 @app.route("/sqs/produce")
 def produce_sqs_message():
     queue = flask_request.args.get("queue", "DistributedTracing")
-    message = "Hello from Python SQS"
+    message = flask_request.args.get("message", "Hello from Python SQS")
```

Comment:
Could you update `docs/weblog` ?

Reply:
Done

Reply:
Ive added all missing endpoints to the docs